python/cugraph/cugraph/dask/structure/mg_property_graph.py

```diff
@@ -331,6 +331,34 @@ def add_vertex_data(self,
                        for n in self.__vertex_prop_dataframe.columns])
         self.__vertex_prop_eval_dict.update(latest)
 
+    def get_vertex_data(self, vertex_ids=None, types=None, columns=None):
+        """
+        Return a dataframe containing vertex properties for only the specified
+        vertex_ids, columns, and/or types, or all vertex IDs if not specified.
+        """
+        if self.__vertex_prop_dataframe is not None:
+            if vertex_ids is not None:
+                df_mask = self.__vertex_prop_dataframe[self.vertex_col_name]\
+                              .isin(vertex_ids)
```

Comment:
Style-wise, I prefer to not use `\`.  Instead, method chaining like this should be done using `(` and `)` as e.g.
```python
                df_mask = (
                    self.__vertex_prop_dataframe[self.vertex_col_name]
                    .isin(vertex_ids)
                )
```
I think this is how `black` would reformat this.

Reply:
Thanks for this suggestion. I made this change and I also prefer it now that I've seen it.  I'll be keeping this in mind in the future.